src/main/java/io/github/pylonmc/pylon/base/items/Elevator.java

```diff
@@ -0,0 +1,126 @@
+package io.github.pylonmc.pylon.base.items;
+
+import com.destroystokyo.paper.event.player.PlayerJumpEvent;
+import io.github.pylonmc.pylon.core.block.BlockStorage;
+import io.github.pylonmc.pylon.core.block.PylonBlock;
+import io.github.pylonmc.pylon.core.block.base.PylonJumpableBlock;
+import io.github.pylonmc.pylon.core.block.base.PylonSneakableBlock;
+import io.github.pylonmc.pylon.core.block.context.BlockCreateContext;
+import io.github.pylonmc.pylon.core.config.Settings;
+import io.github.pylonmc.pylon.core.item.PylonItem;
+import io.github.pylonmc.pylon.core.item.builder.ItemStackBuilder;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.ComponentLike;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.Sound;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.event.player.PlayerToggleSneakEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.persistence.PersistentDataContainer;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static io.github.pylonmc.pylon.base.util.KeyUtils.pylonKey;
+
+
+public class Elevator extends PylonBlock implements PylonSneakableBlock, PylonJumpableBlock {
+
+    public static class Item extends PylonItem {
+
+        public Item(@NotNull ItemStack stack) {
+            super(stack);
+        }
+
+        @Override
+        public @NotNull Map<String, ComponentLike> getPlaceholders() {
+            return Map.of("elevator_range", Component.text(getRange(getKey())));
```

Comment:
```suggestion
            return Map.of("elevator_range", UnitFormat.BLOCKS.format(getRange(getKey())));
```
To be used with my lore suggestion